packages/performance/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/performance/classes/results/GetMetricResult.java

```diff
@@ -0,0 +1,19 @@
+package io.capawesome.capacitorjs.plugins.firebase.performance.classes.results;
+
+import androidx.annotation.NonNull;
+import com.getcapacitor.JSObject;
+
+public class GetMetricResult {
+
+    private final long num;
+
+    public GetMetricResult(@NonNull long value) {
+        this.num = value;
+    }
+
+    public JSObject toJSObject() {
+        JSObject result = new JSObject();
+        result.put("value", this.num == 0 ? null : this.num);
```

Comment:
```suggestion
        result.put("value", this.num);
```

We should not change the behavior of the Firebase SDKs. These plugins are just wrappers.